src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -260,6 +260,29 @@ uct_dc_mlx5_ep_am_short_inline(uct_ep_h tl_ep, uint8_t id, uint64_t hdr,
     return UCS_OK;
 }
 
+static ucs_status_t UCS_F_ALWAYS_INLINE uct_dc_mlx5_ep_am_short_iov_inline(
+        uct_ep_h tl_ep, uint8_t id, const uct_iov_t *iov, size_t iovcnt,
+        size_t iov_length)
+{
+    uct_dc_mlx5_iface_t *iface = ucs_derived_of(tl_ep->iface,
```

Comment:
can we unify more code between rc and dc?
e.g have common short_iov and short_iov_dm functions which will also get the "txqp" as a parameter

Reply:
I'm not sure that I get your point. uct_[rc|dc]_ep_short_iov_dm() call different bcopy functions.
```
uct_rc_mlx5_ep_short_iov_dm()
{
    /* var declaration */
    uct_rc_mlx5_common_dm_make_data(...);
    uct_rc_mlx5_txqp_bcopy_post(...);
}
```
```
uct_dc_mlx5_ep_short_iov_dm()
{
    /* var declaration */
    uct_rc_mlx5_common_dm_make_data(...);
    uct_dc_mlx5_iface_bcopy_post(...);
}
```

Reply:
can we unite these bcopy functions as well?

Reply:
Yes, we can.
`uct_rc_mlx5_txqp_bcopy_post` and `uct_dc_mlx5_iface_bcopy_post` wrap call for `uct_rc_mlx5_txqp_dptr_post` and help to avoid passing redundant arguments. These methods are used in several places. Thus, a new PR should be opened in case of this re-factoring.